### PR TITLE
add release notes for v0.1.10

### PR DESCRIPTION
**Pull request by hengm3467**
Also did some formatting improvements and some rephrasing

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -7,44 +7,100 @@ slug: /release-notes
...
+* Add the default super user “postgres”.  [#3127](https://github.com/singularity-data/risingwave/pull/3127)
+* The default schema name is changed to “public” from “dev”.  [#3166](https://github.com/singularity-data/risingwave/pull/3166)
+
+#### Connectors
+
+* Add random seed for the Datagen Source Connector. [#3124](https://github.com/singularity-data/risingwave/pull/3124)
+
+### Assets
+
+* [risingwave-v0.1.8-x86_64-unknown-linux.tar.gz](https://github.com/singularity-data/risingwave/releases/download/v0.1.8/risingwave-v0.1.10-x86_64-unknown-linux.tar.gz)(Linux)
```
Comment (skyzh) [inline docs/release-notes.md]:
```suggestion
* [risingwave-v0.1.10-x86_64-unknown-linux.tar.gz](https://github.com/singularity-data/risingwave/releases/download/v0.1.10/risingwave-v0.1.10-x86_64-unknown-linux.tar.gz)(Linux)
```

**Review by skyzh — APPROVED** [review]
Rest LGTM

File: docs/release-notes.md [hunk 2]
```diff
@@ -7,44 +7,100 @@ slug: /release-notes
...
+* Intervals multiplying intervals by floats. [#3641](https://github.com/singularity-data/risingwave/pull/3641)
+* Support more temporal operations. [#3472](https://github.com/singularity-data/risingwave/pull/3472)
+* Support Common Table Expressions (CTEs) as input of time window functions. [#3188](https://github.com/singularity-data/risingwave/pull/3188)
+* Add these new functions:
+    * `concat()` for concatenating strings [#3091](https://github.com/singularity-data/risingwave/pull/3091)
+    * `repeat()` for repeating string [#3148](https://github.com/singularity-data/risingwave/pull/3148)
+    * `octet_length()` and bit_length() for getting string length [#3526](https://github.com/singularity-data/risingwave/pull/3526)
+    * `Row()` for constructing rows [#2914](https://github.com/singularity-data/risingwave/pull/2914) #3152 (https://github.com/singularity-data/risingwave/pull/3152)
+    * `pg_typeof()` for getting data types of values [#3494](https://github.com/singularity-data/risingwave/pull/3494)
+    * `current_database()` for listing databases [#3650](https://github.com/singularity-data/risingwave/pull/3650)
```
Comment (yezizp2012) [inline docs/release-notes.md]:
```suggestion
    * `current_database()` for getting the name of current database in the session [#3650](https://github.com/singularity-data/risingwave/pull/3650)
```

Comment (hengm3467) [reply docs/release-notes.md]: @yezizp2012  Thanks. Suggestion accepted with a minor change (adding "the" in front of "current").

**Review by yezizp2012 — APPROVED** [review]

**Review by xiangjinwu — APPROVED** [review]

**Pull request merged**